packages/browser-utils/src/metrics/cls.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  SEMANTIC_ATTRIBUTE_EXCLUSIVE_TIME,
+  SEMANTIC_ATTRIBUTE_SENTRY_MEASUREMENT_UNIT,
+  SEMANTIC_ATTRIBUTE_SENTRY_MEASUREMENT_VALUE,
+  SEMANTIC_ATTRIBUTE_SENTRY_OP,
+  SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN,
+  getClient,
+  getCurrentScope,
+} from '@sentry/core';
+import type { SpanAttributes } from '@sentry/types';
+import { browserPerformanceTimeOrigin, dropUndefinedKeys, htmlTreeAsString, logger } from '@sentry/utils';
+import { DEBUG_BUILD } from '../debug-build';
+import { addClsInstrumentationHandler } from './instrument';
+import { msToSec, startStandaloneWebVitalSpan } from './utils';
+import { onHidden } from './web-vitals/lib/onHidden';
+
+/**
+ * Starts tracking the Cumulative Layout Shift on the current page and collects the value once
+ *
+ * - the page visibility is hidden
+ * - a navigation span is started (to stop CLS measurement for SPA soft navigations)
+ *
+ * Once either of these events triggers, the CLS value is sent as a standalone span and we stop
+ * measuring CLS.
+ */
+export function trackClsAsStandaloneSpan(): void {
+  let standaloneCLsValue = 0;
+  let standaloneClsEntry: LayoutShift | undefined;
+
+  if (!supportsLayoutShift()) {
+    return;
+  }
```

Comment:
Starting with 0 elegantly solves the 0 CLS value problem (see #https://github.com/getsentry/sentry-javascript/pull/13056). However, since this happens outside the web-vitals library, we need to add a browser support check for CLS to avoid sending 0 values on browsers that don't support this.